pages/index.tsx

```diff
@@ -79,6 +79,26 @@ export default function Home() {
       router.push(`/student/classroom/${classroom}`);
     }
   }
+
+  // for dev link shortcut
+  async function testVisitStudentsPage() {
+    const classroom = sampleClassroomName; // using test classroom
+    const classroomObj = getClassroom(classroom);
+    if (!classroomObj) return window.alert(`Invalid classroom: ${classroom}`);
+    const getResponse = await fetch(`${apiUrl}/classrooms/${classroom}`);
+    const { isActive } = await getResponse.json();
+    if (!isActive)
+      return window.alert(
+        `Classroom not activated: ${classroom}\n Please wait for your teacher to activate your classroom and try again.`,
+      );
+    const student = `TestUser${Math.trunc(Math.random() * 1000).toString()}`; // generate random student name
```

Comment:
Looks good! Just a tiny tweak and good to merge.  I would change `TestUser` to `Student` and up the random to `10,000` as Moshe mentioned in his https://github.com/mssiegel/frempco-client/issues/79#issuecomment-1063014259

```suggestion
    const student = `Student ${Math.trunc(Math.random() * 10000).toString()}`; // generate random student name
```

Reply:
Right! Thanks for catching that